docs/guides/cluster-alerts/cert.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Cert
+menu:
+  product_searchlight_6.0.0-rc.0:
+    identifier: guides-cert
+    name: Cert
+    parent: cluster-alert
+    weight: 20
+product_name: searchlight
+menu_name: product_searchlight_6.0.0-rc.0
+section_menu_id: guides
+---
+
+> New to Searchlight? Please start [here](/docs/concepts/README.md).
+
+# Check cert
+
+Check command `cert` checks the expiration timestamp of any certificate from Secrets. No longer you have to get a surprise that your certificates have expired.
+
+## Spec
+`cert` check command has the following variables:
+
+- `selector` - Selector (label query) to filter on, supports '=', '==', and '!='
+- `secretName` - Name of secret from where certificates are checked
+- `secretKey` - Name of secret key where certificates are kept
+- `warning` - Remaining duration for Warning state. [Default: 360h]
+- `critical` - Remaining duration for Critical state. [Default: 120h]
+
+Execution of this command can result in following states:
+
+- OK
+- Warning
+- Critical
+- Unknown
+
+
+## Tutorial
+
+### Before You Begin
+At first, you need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [Minikube](https://github.com/kubernetes/minikube).
+
+Now, install Searchlight operator in your cluster following the steps [here](/docs/setup/install.md).
+
+To keep things isolated, this tutorial uses a separate namespace called `demo` throughout this tutorial. Run the following command to prepare your cluster for this tutorial:
+
+```console
+$ kubectl create namespace demo
+namespace "demo" created
+
+$ kubectl get namespaces
+NAME          STATUS    AGE
+default       Active    6h
+kube-public   Active    6h
+kube-system   Active    6h
+demo          Active    4m
+```
+
+### Create Alert
+In this tutorial, we are going to create an alert to check `cert`.
+
+```yaml
+$ cat ./docs/examples/cluster-alerts/cert/demo-0.yaml
+
+apiVersion: monitoring.appscode.com/v1alpha1
+kind: ClusterAlert
+metadata:
+  name: cert-demo-0
+  namespace: demo
+spec:
+  check: cert
```

Comment:
Create a Secret with a TLS cert and then use that name to check for expiration. Use onessl to issue the cert.
https://github.com/kubepack/onessl/releases/tag/0.3.0